pydaikin/daikin_airbase.py

```diff
@@ -241,7 +239,15 @@ async def set_zone(self, zone_id, key, value):
         }
 
         if self.support_zone_temperature:
+            params.update({"lztemp_c": self.values["lztemp_c"]})
             params.update({"lztemp_h": self.values["lztemp_h"]})
 
-        _LOGGER.debug("Sending request to %s with params: %s", path, params)
-        await self._get_resource(path, params)
+        # # Convert params dictionary to query string format
+        params_str = "&".join(f"{k}={v}" for k, v in params.items())
+        path = f"{path}?{params_str}"  # Append params as query string to path
+
+        _LOGGER.debug(
+            "Updating ['aircon/set_zone_setting']: %s",
+            ",".join(f"{k}={unquote(v).replace(" ", "")}" for k, v in params.items()),
```

Comment:
```suggestion
            ",".join(f"{k}={unquote(v).replace(' ', '')}" for k, v in params.items()),
```

You can not use `"` within a func string enclosed by `"` . Should we really replace all spaces??

Reply:
Was just trying to output something readable - the default for mine is `      Zone1` but yea `Bed 1` would break so maybe we shouldn't